src/crc32c.c

```diff
@@ -341,6 +345,8 @@ static crc32c_func_t crc32c_dispatch(unsigned long hwcap)
 // For ifdef detection below
 #    define crc32c_dispatch() crc32c_dispatch(getauxval(AT_HWCAP))
 #    define crc32c_dispatch_ifunc "crc32c_dispatch"
+#  else
+#  warning CRC32 feature detection not implemented for this OS. Falling back to software version.
```

Comment:
Hmm, time to rebase my local crc32 branch for 32bit arm I guess... ;-p

Reply:
Isn't thins whole thing guarded by _CPU_AARCH64_? Not sure I understand the comment.

Reply:
Yeah, I though thihs ifdef was one level up (see edit...)